.gitignore

```diff
@@ -18,4 +18,7 @@
 .idea
 
 # Cmds being built
-cmd/nsm/nsm
\ No newline at end of file
+cmd/nsm/nsm
```

Comment:
Since you are changing gitignore, maybe 2 other binaries can be added? nsm-init and nse??

Reply:
Done